cuqipy_fenics/utilities.py

```diff
@@ -4,15 +4,35 @@
 from numbers import Number
 from cuqi.samples import Samples
 
+# ufl lazy loading
+class _LazyUFLLoader:
```

Comment:
This class has some strange formatting and spacing it seems. Can you sent it through e.g. black?

Reply:
True 👍 I reformated it.